src/index.html

```diff
@@ -7,8 +7,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="defalit"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&family=Nothing+You+Colid+Do&family=Roboto:ital,wght@0,500;0,700;1,500;1,700&family=Satisfy&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo_logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a class="nav__a is-active" href="#apple">Apple</a></li>
```

Comment:
fix this formatting everywhere
![image](https://user-images.githubusercontent.com/70009205/208705124-54d979f4-0a77-4167-a3c9-8c037d411427.png)


```suggestion
          <li>
            <a class="nav__a is-active" href="#apple">Apple</a>
          </li>
```